data/xml/2022.digitam.xml

```diff
@@ -39,22 +41,26 @@
-      <url hash="a034b762">2022.digitam-1.4</url>
+      <url hash="0ea54dc9">2022.digitam-1.3</url>
       <bibkey>kindt-van-elverdinghe-2022-describing</bibkey>
+      <revision id="1" href="2022.digitam-1.3v1" hash="ef7d09ef"/>
+      <revision id="2" href="2022.digitam-1.3v2" hash="0ea54dc9" date="2022-11-04">Updated the pdf with the final version.</revision>
+    </paper>
+    <paper id="4">
+      <title>Analyse Automatique de l’Ancien Arménien. Évaluation d’une méthode hybride « dictionnaire » et « réseau de neurones » sur un Extrait de l’Adversus Haereses d’Irénée de Lyon</title>
+      <author><first>Bastien</first><last>Kindt</last></author>
+      <author><first>Gabriel</first><last>Kepeklian</last></author>
+      <pages>20–27</pages>
+      <abstract>The aim of this paper is to evaluate a lexical analysis (mainly lemmatization and POS-tagging) of a sample of the Ancient Armenian version of the Adversus Haereses by Irenaeus of Lyons (2nd c.) by using hybrid approach based on digital dictionaries on the one hand, and on Recurrent Neural Network (RNN) on the other hand. The quality of the results is checked by comparing data obtained by implementing these two methods with data manually checked. In the present case, 98,37% of the results are correct by using the first (lexical) approach, and 74,64% by using the second (RNN). But, in fact, both methods present advantages and disadvantages and argue for the hybrid method. The linguistic resources implemented here are jointly developed and tested by GREgORI and Calfa.</abstract>
+      <url hash="dbb3c419">2022.digitam-1.4</url>
+      <bibkey>kindt-kepeklian-2022-analyse</bibkey>
+      <revision id="1" href="2022.digitam-1.4v1" hash="a034b762"/>
+      <revision id="2" href="2022.digitam-1.4v2" hash="dbb3c419" date="2022-11-04">Updated the pdf with the final version.</revision>
     </paper>
```

Comment:
If the papers were just switched, then they're not revisions. Just check the main volume, and switch either the metadata or the PDFs, based on which is correct.

Reply:
From the full volume, the PDFs are correct, the metadata is wrong. So revert the revisions, and then switch the metadata order (paper 3 is 4, 4 is 3, make sure the page numbers are correct).

https://aclanthology.org/2022.digitam-1.0.pdf